app/components/CommitFundsUnknownUserAftermath.tsx

```diff
@@ -0,0 +1,99 @@
+import { BigNumber } from "bignumber.js";
+import { TextField } from "redux-form-material-ui";
+
+import { Moment } from "moment";
+import * as React from "react";
+
+import { compose } from "redux";
+import { Field, InjectedFormProps, reduxForm } from "redux-form";
+import { ethereumAddressValidator } from "../validators/ethereumAddressValidator";
+import * as styles from "./Aftermath.scss";
+import { UnderlinedLink } from "./UnderlinedLink";
+
+interface ICommitFundsUnknownUserAftermathProps {
+  lockedAmount?: BigNumber;
+  unlockDate?: Moment;
+  neumarkBalance?: BigNumber;
+}
+
+interface ICommitFundsUnknownUserAftermathForm {
+  address: string;
+}
+
+const estTextFieldStyles = {
+  inputStyle: {
+    color: "#BBC2C7",
+    fontWeight: 500 as 500,
+  },
+  style: {
+    width: "370px",
+    marginLeft: "5px",
```

Comment:
I think we don't margins here. On design they are aligned to left with rest of text lines